### DISCUSSION TITLE
Vulnérabilité CVE-2022-32527.js

### DISCUSSION THREAD

**Discussion by Shygeek64** [community/community#134350] [New to GitHub] [answered]
_<<!DOCTYPE HTML>
<html lang="fr">
<head>
    <meta charset="UTF-8">
    <meta HTTP-equiv="X-UA-Compatible" content="IE=edge">
    <meta name="viewport" content="width-device-width, initial-scale="1.0">
    <title>Document</title>
    <script src="script.js">let numberCve = prompt(CVE-2022-32527) console.log(CVE-2022-32527)</script>
</head>
<body>
    <header id="mainheader">
        {"color="red}
        {"bgcolor="midnight-blue"}
        {"font-size="30%"}
        {"align="center"}
        {"text="strong"}
        {"police-text="time-new-roman"}

        <p>Vector String CVSS: 3.1/AV:N/AC:L/PR:N/UI:N/S:U/C:H/I:H/A:H
            Severity Score CVSS: Critical 9.8 By timidhacker@hackerone.com</p>
            <h1>Name Of CWE: Classic Buffer Overflow</h1>
            <h2>CWE ID: CWE-120</h2>
            <h3>Asset's Name Or Bug Bounty's Prime: Source Code For $690 069,00</h3>
            <h4>EPSS: 0,40%</h4>
            <h5>NVD Published Date: 30/01/2023</h5>
            <h6>NVD Last Modified: 08/02/2023</h6>
            <h7>Reported By Shneider Electric SE</h7>
            <h8>Programmed By HackerOne</h8>
            <h9>To Share With NIST</h9>
        </header>
    </body>
</html>!DOCTYPE HTML>
<html lang="fr">
<head>
    <meta charset="UTF-8">
    <meta HTTP-equiv="X-UA-Compatible" content="IE=edge">
    <meta name="viewport" content="width-device-width, initial-scale="1.0">
    <title>Document</title>
    <script src="script.js">let numberCve = prompt(CVE-2022-32527) console.log(CVE-2022-32527)</script>
</head>
<body>
    <header id="mainheader">
        {"color="red}
        {"bgcolor="midnight-blue"}
        {"font-size="30%"}
        {"align="center"}
        {"text="strong"}
        {"police-text="time-new-roman"}

        <p>Vector String CVSS: 3.1/AV:N/AC:L/PR:N/UI:N/S:U/C:H/I:H/A:H
            Severity Score CVSS: Critical 9.8 By timidhacker@hackerone.com</p>
            <h1>Name Of CWE: Classic Buffer Overflow</h1>
            <h2>CWE ID: CWE-120</h2>
            <h3>Asset's Name Or Bug Bounty's Prime: Source Code For $690 069,00</h3>
            <h4>EPSS: 0,40%</h4>
            <h5>NVD Published Date: 30/01/2023</h5>
            <h6>NVD Last Modified: 08/02/2023</h6>
            <h7>Reported By Shneider Electric SE</h7>
            <h8>Programmed By HackerOne</h8>
            <h9>To Share With NIST</h9>
        </header>
    </body>
</html>_

**Comment by Shygeek64** [comment, marked as answer]
WTF***?!? Bravo!!! on ne savait pas qu'on avait une (CEH) avec nous maintenant pour ta vulnérab, on va maintenant passer aux choses sérieuses: Par où commencer?

**Reply by Shygeek64** [reply]
Ou devrais-je dire WTH***?!? Pour changer la phrase, qu'elle soit plus nickel... What The Hack 👍

**Comment by Shygeek64** [comment]
Est-ce que quelqu'un peut me dire comment effacer et recommencer svp ? Merci

**Reply by Sotot** [reply]
# l' ..

**Comment by Lucky113114** [comment]
Ghnvjk

**Comment by Jytvs** [comment]
> _<
> 
> <title>Document</title> <script src="script.js">let numberCve = prompt([CVE-2022-32527](https://github.com/advisories/GHSA-hfrp-5qw2-x2qm)) console.log([CVE-2022-32527](https://github.com/advisories/GHSA-hfrp-5qw2-x2qm))</script> {"color="red} {"bgcolor="midnight-blue"} {"font-size="30%"} {"align="center"} {"text="strong"} {"police-text="time-new-roman"}
> ```
>     <p>Vector String CVSS: 3.1/AV:N/AC:L/PR:N/UI:N/S:U/C:H/I:H/A:H
>         Severity Score CVSS: Critical 9.8 By timidhacker@hackerone.com</p>
>         <h1>Name Of CWE: Classic Buffer Overflow</h1>
>         <h2>CWE ID: CWE-120</h2>
>         <h3>Asset's Name Or Bug Bounty's Prime: Source Code For $690 069,00</h3>
>         <h4>EPSS: 0,40%</h4>
>         <h5>NVD Published Date: 30/01/2023</h5>
>         <h6>NVD Last Modified: 08/02/2023</h6>
>         <h7>Reported By Shneider Electric SE</h7>
>         <h8>Programmed By HackerOne</h8>
>         <h9>To Share With NIST</h9>
>     </header>
> </body>
> ```
> 
> !DOCTYPE HTML> <title>Document</title> <script src="script.js">let numberCve = prompt([CVE-2022-32527](https://github.com/advisories/GHSA-hfrp-5qw2-x2qm)) console.log([CVE-2022-32527](https://github.com/advisories/GHSA-hfrp-5qw2-x2qm))</script> {"color="red} {"bgcolor="midnight-blue"} {"font-size="30%"} {"align="center"} {"text="strong"} {"police-text="time-new-roman"}
> ```
>     <p>Vector String CVSS: 3.1/AV:N/AC:L/PR:N/UI:N/S:U/C:H/I:H/A:H
>         Severity Score CVSS: Critical 9.8 By timidhacker@hackerone.com</p>
>         <h1>Name Of CWE: Classic Buffer Overflow</h1>
>         <h2>CWE ID: CWE-120</h2>
>         <h3>Asset's Name Or Bug Bounty's Prime: Source Code For $690 069,00</h3>
>         <h4>EPSS: 0,40%</h4>
>         <h5>NVD Published Date: 30/01/2023</h5>
>         <h6>NVD Last Modified: 08/02/2023</h6>
>         <h7>Reported By Shneider Electric SE</h7>
>         <h8>Programmed By HackerOne</h8>
>         <h9>To Share With NIST</h9>
>     </header>
> </body>
> ```
> 
> _

Bank BPI#000649420157
Branch marilao 0064
Manila philipines
Name jeffry acohon

**Comment by Jytvs** [comment]
BPI bank 000649420157
Branch 0064 marilao
Manila philipines
Name jeffry acohon

**Comment by ghostinhershell** [comment]
Hi @Shygeek64, thanks for participating!

We take our [[Code of Conduct](https://github.com/community/community/blob/920a928ffe3ae4863b64a4ea26917046e11f5c1a/CODE_OF_CONDUCT.md)](https://github.com/community/community/blob/0ca37cf594e438fd4aec3c7f3736b95e6b0b3567/CODE_OF_CONDUCT.md) very seriously and want to help ensure that everyone has a good experience free of antagonism and harassment. Unfortunately, we don’t currently have moderators for languages other than English. Until that changes, we need to ask that everyone use English here in the GitHub Community when posting. We’ll be locking any posts in languages other than English for now, including this one.
